docs/en-US/component/mention.md

```diff
@@ -0,0 +1,145 @@
+| offset                               | offset of the dropdown panel                                                           | ^[number]                                                                    | `0`        |
+| whole                                | when backspace is pressed to delete, whether the mention content is deleted as a whole | ^[boolean]                                                                   | `false`    |
+| check-is-whole                       | when backspace is pressed to delete, check if the mention is a whole                   | ^[Function]`(pattern: string, prefix: string) => boolean`                    | -          |
+| loading                              | whether the dropdown panel of mentions is in a loading state                           | ^[boolean]                                                                   | `false`    |
+| model-value / v-model                | input value                                                                            | ^[string]                                                                    | -          |
+| popper-class                         | custom class name for dropdown panel                                                   | ^[string]                                                                    | -          |
+| popper-options                       | [popper.js](https://popper.js.org/docs/v2/) parameters                                 | ^[object] refer to [popper.js doc](https://popper.js.org/docs/v2/)           | -          |
+| [input props](./input.md#attributes) | -                                                                                      | -                                                                            | -          |
+
+### Events
+
+| Name                              | Description                         | Type                                                         |
+| --------------------------------- | ----------------------------------- | ------------------------------------------------------------ |
+| search                            | trigger when prefix hit             | ^[Function]`(pattern: string, prefix: string) => void`       |
+| select                            | trigger when user select the option | ^[Function]`(option: MentionOption, prefix: string) => void` |
+| [input events](./input.md#events) | -                                   | -                                                            |
+
+### Slots
+
+| Name                            | Description                           |
+| ------------------------------- | ------------------------------------- |
+| label                           | content as option label               |
+| loading                         | content as option loading             |
+| header                          | content at the top of the dropdown    |
+| footer                          | content at the bottom of the dropdown |
```

Comment:
`slot` should also be labeled with a type, indicating the acceptable parameter values ​​and types.

Reply:
I see that other components do not have parameter types marked for solt. Is there any reference?

Reply:
https://element-plus.org/en-US/component/table.html#table-column-slots

Reply:
I found it. [upload slots](https://element-plus.org/zh-CN/component/upload.html#%E6%8F%92%E6%A7%BD)